lib/plugin.dart

```diff
@@ -331,7 +319,7 @@ class AtomDartPackage extends AtomPackage {
       // show infos and todos
       'configureErrorsView': {
         'title': "Errors view configuration",
-        'description': 'Choose which types of items to show in the errors view.',
+        'description': 'Choose which types of items to show in the Linter.',
```

Comment:
That's a bit tricky to answer, `linter` is "just" the package that manages all the providers and processes their results into a unified set. Built on top of that are UIs that display those results to users, `linter-ui-default` being the bundled one.

Personally I'd probably just go with "... show in Linter" (ie. remove the `the`), since the full UIs out there display themselves as that.